internal/jimm/jimm.go

```diff
@@ -111,6 +114,25 @@ type RoleManager interface {
 	CountRoles(ctx context.Context, user *openfga.User) (int, error)
 }
 
+// GroupManager provides a means to manage groups within JIMM.
+type GroupManager interface {
```

Comment:
I thought we weren't doing interfaces for now? If we are, can you add one for roles too please.

Reply:
Nvm gotten confused